src/Core/Application/ValueResolver/ConfigurableAbstractValueResolver.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\ValueResolver;
+
+use SprykerSdk\SdkContracts\ValueResolver\ConfigurableValueResolverInterface;
+
+abstract class ConfigurableAbstractValueResolver extends AbstractValueResolver implements ConfigurableValueResolverInterface
+{
+    /**
+     * @var mixed
+     */
+    protected $defaultValue;
```

Comment:
The resolver should not have a state.

Reply:
What do you mean? it's configuration

Reply:
I mean this is a business model. It must be stateless.